packages/reakit/src/Dialog/__utils/useEventListenerOutside.ts

```diff
@@ -20,11 +20,7 @@ function dialogContains(target: Element) {
 }
 
 function isDisclosure(target: Element, disclosure: HTMLElement) {
-  if (contains(disclosure, target)) return true;
-  return (
-    disclosure.id &&
-    disclosure.id === target.getAttribute?.("aria-activedescendant")
-  );
+  return contains(disclosure, target);
 }
```

Comment:
Looks like this is not necessary anymore as now we're always capturing the right disclosure element, and not the container with `aria-activedescendant`. But I'll keep an eye on this.